docs/api.md

```diff
@@ -2595,7 +2595,7 @@ If the target is not of type `"page"`, returns `null`.
 #### target.type()
 - returns: <[string]>
 
-Identifies what kind of target this is. Can be `"page"`, `"service_worker"`, `"browser"` or `"other"`.
```

Comment:
As this list grows, we should probably provide some explaining links as to what these types are: [`"background_page"`](https://developer.chrome.com/extensions/background_pages)